lib/internal/util/inspect.js

```diff
@@ -660,12 +657,10 @@ function getPrefix(constructor, tag, fallback, size = '') {
 // Look up the keys of the object.
 function getKeys(value, showHidden) {
   let keys;
-  const symbols = ObjectGetOwnPropertySymbols(value);
   if (showHidden) {
-    keys = ObjectGetOwnPropertyNames(value);
-    if (symbols.length !== 0)
-      keys.push(...symbols);
+    keys = ReflectOwnKeys(value);
   } else {
+    const symbols = ObjectGetOwnPropertySymbols(value);
```

Comment:
Maybe we can try to revert this part and keep only the change in `addPrototypeProperties`.

Reply:
Done in <https://github.com/nodejs/node/commit/6c885555951158a910522b87c0b97b4c8f1f365a>